File: 119-pascals-triangle-ii/pascals-triangle-ii.java
class Solution {
    public List<Integer> getRow(int rowIndex) {
        List<Integer> prev=new ArrayList<>();
        for(int i=0;i<=rowIndex;i++){
            List<Integer> list=new ArrayList<>();
            for(int j=0;j<=i;j++){
                if(j==0 || j==i)
                    list.add(1);
                else
                    list.add(prev.get(j) + prev.get(j-1));
            }
            prev=list;
        }
        return prev;
    }
}

File: 119-pascals-triangle-ii/pascals-triangle-ii.js
/**
 * @param {number} rowIndex
 * @return {number[]}
 */
var getRow = function(rowIndex) {
    var outputArray = [];
    for(var i = 0 ; i<= rowIndex; i++) { 
        var temp = [] ;
        for(var j = 0; j <= i ; j++ ){ 
            if( j == 0 || j == i)
                temp.push(1);
            else{
                temp.push( outputArray.at(j) + outputArray.at(j-1));
            }
        }
        outputArray = temp;
    }
    return outputArray;
};